1907-05-20.xml

```diff
@@ -0,0 +1,1602 @@
+                            Wednesdays at 5 p.m. for Port Sudan and Suakim in connection with the
+                            Sudan Government Railways express service to Khartoum.</p>
+                        <p><hi rend="bold">RED SEA COASTING MAIL SERVICE.</hi></p>
+                        <p>Mail steamers leave Suez on Mondays at 5 p.m. for TOR (for Sinai. YAMBO
+                            and JEDDAH, and continuing in the alternate weeks to PORT SUDAN, SUAKIM,
+                            MASSOWAH, HODEIDAH and ADEN.</p>
+                        <p>For further information refer to the Co.’s Agencies at Alexandria, Cairo,
+                            Port Said, Suez, or at THOS. COOK &amp; SON (Egypt) Ltd.,
+                            HAMBURG-AMERIKA REISE-BUREAU or other Tourist Agencies.</p>
+                    </div>
+                    <div type="advert" xml:id="deg-ad-dll01">
+                        <head>Deutsche Levante-Linie.</head>
+                        <p>Mail and Passenger Steamships. Regular Service fortnightly from HAMBURG,
+                            weekly from ANTWERP; every 4 weeks from BORDEAUX direct to ALEXANDRIA;
+                            homewards every 2 weeks from ALEXANDRIA for ROTTERDAM and HAMBURG. Goods
+                            forwarded at through-rates from all German Railway Stations on direct
+                            Bills of Lading to ALEXANDRIA, CAIRO, SYRIA, etc. Special facilities for
+                            conveyance of cotton, etc from ALEXANDRIA via ROTTERDAM and HAMBURG to
+                            German manufacturing towns. </p>
+                        <p>EXPECTED AT ALEXANDRIA.</p>
+                        <p><name>S.S. Enos</name>
+                            <date when="1905-07-20">May 15</date> from <placeName>Hamburg &amp;
+                                Bourdeaux.</placeName></p>
+                        <p><name>S.S. Athos</name>
+                            <date when="1905-05-16">May 16</date> from
+                                <placeName>Hamburg</placeName> bound for <placeName>Rotterdamn &amp;
+                                Hamburg.</placeName></p>
+                        <p><name>S.S. Seriphos</name>
+                            <date when="1905-07-31">May 17</date> from
+                                <placeName>Antwerp.</placeName></p>
+                        <p><name>S.S. Andros</name>
+                            <date>May 25</date> from <placeName> Hamburg</placeName></p>
+                        <p>The S.S. Rhodes now in port loading, will sail for Rotterdam &amp;
+                            Hamburg on or about the 15th inst.</p>
+                        <p>For tariff and particulars apply to ADOLPHE STROSS, Alexandria,
+                            Agent.</p>
+                        <p><measure type="indexNo">15-9-907</measure></p>
+                    </div>
+                </div>
```

Comment:
move this `</div>` tag, which closes the section, to just after the last ad on the page.